packages/playback-core/src/text-tracks.ts

```diff
@@ -137,6 +149,9 @@ export function addTextTrack(
   if (id) {
     trackEl.id = id;
   }
+  if (!!defaultTrack) {
+    trackEl.default = true;
+  }
```

Comment:
```suggestion
  trackEl.default = !!defaultTrack
```

Reply:
I'm intentionally only setting `default` if `true` here (due to **_default_** value of `default`), but I get the sentiment.